EXILED/Exiled.CustomItems/Commands/Give.cs

```diff
@@ -97,27 +98,44 @@ public bool Execute(ArraySegment<string> arguments, ICommandSender sender, out s
                     response = $"Custom item {item?.Name} given to all players who can receive them ({eligiblePlayers.Count} players)";
                     return true;
                 default:
-                    if (Player.Get(identifier) is not { } player)
-                    {
-                        response = $"Unable to find player: {identifier}.";
-                        return false;
-                    }
+                    break;
+            }
 
-                    if (!CheckEligible(player))
-                    {
-                        response = "Player cannot receive custom items!";
-                        return false;
-                    }
+            string[] newargs;
+            List<ReferenceHub> list = RAUtils.ProcessPlayerIdOrNamesList(arguments, 1, out newargs);
```

Comment:
you could use Player.GetProcessedData(argument..)

Reply:
how come? is it to avoid creating a var that wont get used?

Reply:
to not need the way to do multiple Player.Get

Reply:
https://github.com/icedchai/EXILED/pull/1

Reply:
okay, makes sense